### PR TITLE
Update beam flux from config J to K

### PR DESCRIPTION
**Pull request by VCLanNguyen**
Update from flux config J-v1 to config K-v1 
What’s new: 
New flux production on gpvm
Kaon weight is applied with a stand alone code and does not have MiniBooNE dependency 

Validation can be found on docdc: [32091](https://sbn-docdb.fnal.gov/cgi-bin/sso/ShowDocument?docid=32091)

### CODE REVIEW

**Comment by VCLanNguyen** [comment]
trigger

**Comment by RachelCoackley** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_75_02 SBNSoftware/sbncode@v09_75_02 SBNSoftware/sbnanaobj@v09_21_02 SBNSoftware/sbnobj@v09_17_03

**Comment by RachelCoackley** [comment]
The CI tests are coming up failed due to nucosmics_gen_quick_test_sbndcode having an error in data production. I ran the CI tests again (https://dbweb8.fnal.gov:8443/LarCI/app/ns:sbnd/build_detail/phase_details?build_id=sbnd_ci/12217&platform=Linux%203.10.0-1160.88.1.el7.x86_64&phase=ci_tests&buildtype=slf7%20e20:prof) and got the same result. It looks like it's not being able to find a flux file, @VCLanNguyen would you be able to have a quick look? Thanks!

File: sbndcode/LArSoftConfigurations/gen/genie_sbnd.fcl [hunk 1]
```diff
@@ -190,12 +190,28 @@ sbnd_flux_bnb_nu_Jv1_test1: {
...
+#
+sbnd_flux_bnb_nu_Kv1: {
+  FluxType:       "simple_flux"
+  FluxCopyMethod: "DIRECT"
+  FluxSearchPaths: "/cvmfs/sbnd.osgstorage.org/pnfs/fnal.gov/usr/sbnd/persistent/stash/fluxFiles/bnb/BooNEtoGSimple/configK-v1/july2023/neutrinoMode"
+  FluxFiles: [ "gsimple_april07_baseline_*_redecay_wkaonwgh.root" ]
+}
+
+sbnd_flux_bnb_nu_Kv1_test1: {
+  @table::sbnd_flux_bnb_nu_Jv1
```
Comment (marcodeltutto) [inline sbndcode/LArSoftConfigurations/gen/genie_sbnd.fcl:204]: This should have "K", no "J"

**Review by marcodeltutto — CHANGES_REQUESTED** [review]

**Comment by VCLanNguyen** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v09_75_02 SBNSoftware/sbncode@v09_75_02 SBNSoftware/sbnanaobj@v09_21_02 SBNSoftware/sbnobj@v09_17_03

File: sbndcode/LArSoftConfigurations/gen/genie_sbnd.fcl (hunk 1 above)
Comment (VCLanNguyen) [reply sbndcode/LArSoftConfigurations/gen/genie_sbnd.fcl]: fixed the typo!

**Review by marcodeltutto — APPROVED** [review]

**Comment by VCLanNguyen** [comment]
CI picked up fcl changes 

```
84: ===================================================================================================================================
85: 38c38
86: <             "gsimple_bnb_neutrino_sbnd_v2_wvtxt_20210322_*.root"
87: ---
88: >             "gsimple_april07_baseline_*_redecay_wkaonwgh.root"
89: 40c40
90: <          FluxSearchPaths: "/cvmfs/sbnd.osgstorage.org/pnfs/fnal.gov/usr/sbnd/persistent/stash/fluxFiles/bnb/BooNEtoGSimple/configJ-v1/feb2023/neutrinoMode/"
91: ---
92: >          FluxSearchPaths: "/cvmfs/sbnd.osgstorage.org/pnfs/fnal.gov/usr/sbnd/persistent/stash/fluxFiles/bnb/BooNEtoGSimple/configK-v1/july2023/neutrinoMode"
93: ===================================================================================================================================
```

**Comment by VCLanNguyen** [comment]
Expected [nucosmics_caf_quick_test_sbndcode](https://dbweb8.fnal.gov:8443/LarCI/app/ns:SBND/build_detail/test_details?build_id=sbnd_ci/12219&platform=Linux%203.10.0-1160.88.1.el7.x86_64&buildtype=slf7%20c7:prof&phase=ci_tests&test=nucosmics_caf_quick_test_sbndcode) to be different given new flux files so probably different nu interactions

**Comment by fjnicolas** [comment]
Thanks @VCLanNguyen! All looks good from my part. I'm assuming you mean [nucosmics_gen_quick_test_sbndcode](https://dbweb8.fnal.gov:8443/LarCI/app/ns:SBND/build_detail/test_details?build_id=sbnd_ci/12220&platform=Linux%203.10.0-1160.88.1.el7.x86_64&buildtype=slf7%20e20:prof&phase=ci_tests&test=nucosmics_gen_quick_test_sbndcode)

**Pull request merged**